schema/codelists/partyRole.csv

```diff
@@ -1,21 +1,21 @@
 Code,Title,Description,Source,Deprecated
 buyer,Buyer,"The organization aiming to conclude a contract with a supplier or to use the goods, works or services resulting from the contract.",,
-procuringEntity,Procuring entity,"The organization managing the contracting process. If an organization is both a buyer and a procuring entity (as can be the case in simple contracting processes), its roles should include 'buyer', but not 'procuringEntity'.",,
+procuringEntity,Procuring entity,"The organization managing the contracting (or planning) process. If an organization is both a buyer and a procuring entity (as can be the case in simple contracting processes), its roles should include 'buyer', but not 'procuringEntity'.",,
 supplier,Supplier,An organization with which a buyer or a procuring entity decided to conclude a contract.,,
 tenderer,Tenderer,An organization that submitted a bid.,,
 funder,Funder,The funder is an organization providing money or finance for this contracting process.,,1.2
-enquirer,Enquirer,An organization that has made an enquiry during the enquiry phase of a contracting process.,,
+enquirer,Enquirer,An organization that has made an enquiry during the enquiry phase of a contracting (or planning) process.,,
```

Comment:
Do planning processes have an enquiry phase?

Reply:
Not sure. I don't really have an "enquiry phase" in my mental model of contracting (which is quite possibly my fault because questions don't have fields in eForms and they are not really part of the Directives, so I don't know much about them).

For me, the only thing linked to questions is a deadline (typically a few weeks before bid submission, with the reasoning being that tenderers need time to reflect the answers in their bids) and then the modalities of how/where to send them. I don't really think of a deadline only after which you can _start_ asking questions. 

Obviously, you will be asking different / more concrete questions after the procurement documents have been published (contracting process, e.g. "could you clarify sentence XYZ") than before (planning process, e.g. "could you clarify vague paragraph XYZ in the PIN", "are you aware of solution X that would solve your need but might end up not being applicable in your future contracting process because of thing ABC that you mention in your planning process" or perhaps even "do you have any news on when the contracting process will be launched"?), but for me both seem similar enough to group them under one code.

In this PR, I would leave it as is, and then deal with it in more detail in a follow-up issue: https://github.com/open-contracting/standard/issues/1522

Reply:
I prefer to not include the change in this PR. If needed, we can do the change in #1522.

Reply:
Ok. Removed in https://github.com/open-contracting/standard/pull/1513/commits/c5d811a90b71818b6ee980d521154e03dae91a18